src/main/java/ch/njol/skript/conditions/CondCanAge.java

```diff
@@ -0,0 +1,39 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Breedable;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Can Age")
+@Description("Returns whether or not an entity will be able to age/grow up.")
```

Comment:
```suggestion
@Description("Checks whether or not an entity is able to age/grow up.")
```
"Check" is a better word for this